### ISSUE TITLE
Allow flameshot to work in a "one off mode"

### ISSUE THREAD

**Issue by Robbendebiene** [flameshot-org/flameshot#1716]
**TLDR:** Expose one (or two?) settings to automatically open the flameshot UI on launch (basically a setting for `flameshot gui`) and close flameshot after the screenshot has been taken.

**Explanation:**
Windows and GNOME screenshot tools work like this:
1. **Clicking the app launcher starts the app window to take screenshots**
2. Click the take screenshot button to initiate a screenshot
3. Select the region
4. Screenshot is taken
5. **Screenshot app is closed**

Flameshot is superior in functionality and UI. I like that it doesn't open a window where you first have to click an extra button in order to take a screenshot like the other apps I've mentioned above. However using the systray icon in order to trigger a screenshot requires more work then clicking a pinned app launcher icon (1 click for systray + mouse movement + 1 click for "take a screenshot" VS 1 click on app launcher).

The behaviour I have in mind may best be described as a "one off mode".

1. Click on flameshot app launcher automatically opens the screenshot overlay/ui (flameshot gui -d 200)
2. Use cool flamshot tools and finish your screenshot (for example by copying it)
3. Automatically close flameshot

Is there a chance a PR would be accepted exposing this as an additional setting?

--------------

Side note: I first wanted to comment on https://github.com/flameshot-org/flameshot/issues/1696 but because it has already been closed and my request slightly differs I decided to open a new one. (please forgive me)

**Comment by borgmanJeremy** [comment]
What platform are you on? The reason I ask is as long as flameshot is running you should be able to customize the app launcher to do anything (including running flameshot gui). You also don't need to use the systray at all, simply hitting PrintScreen will take a screen shot in flameshot as long as the daemon is running. 

> Is there a chance a PR would be accepted exposing this as an additional setting?

Not as currently written. The correct way to do what you are asking is to make DBus an optional dependency for CLI parsing. I would like to do this anyway to allow CLI parsing to work on Windows and Mac (#1579).

**Comment by Robbendebiene** [comment]
> What platform are you on?

I'm on Linux (Manjaro GNOME).

> You also don't need to use the systray at all, simply hitting PrintScreen will take a screen shot in flameshot as long as the daemon is running.

I'm aware of this option, but I'm still using the system build in screenshot functionality for the print key.

**Comment by borgmanJeremy** [comment]
Im more familiar with KDE, but in KDE I can bind anything I want (like CTRL + P), and have that kick off the flameshot gui command. Does gnome have something like that?

**Comment by Robbendebiene** [comment]
Yes Gnome supports this as well via the settings app.
